tests/ValidatorTest.php

```diff
@@ -26,83 +23,6 @@ public function testInvalidRuleClassShouldThrowComponentException()
         $this->setExpectedException('Respect\Validation\Exceptions\ComponentException');
         Validator::iDoNotExistSoIShouldThrowException();
     }
-    public function testSetTemplateWithSingleValidatorShouldUseTemplateAsMainMessage()
-    {
-        try {
-            Validator::callback('is_int')->setTemplate('{{name}} is not tasty')->assert('something');
-        } catch (NestedValidationExceptionInterface $e) {
-            $this->assertEquals('"something" is not tasty', $e->getMainMessage());
-        }
```

Comment:
If no exception was thrown, the test was succeed anyway.

The same happens to the tests below.